### PR TITLE
add linter rule for no focussed tests

### PR DESCRIPTION
**Pull request by johnduffell**
Seems like just about everyone has left a test with fdescribe in at some stage.  This is really bad because it will basically stop running tests in the automation!  So I've added a linter with that rule in it https://www.npmjs.com/package/eslint-plugin-jasmine

not sure why so many changes to npm-shrinkwrap

Seems to work but I feel like I need a test for it...

@OliverJAsh @jbreckmckye

### CODE REVIEW

**Comment by OliverJAsh** [comment]
`npm shrinkwrap` is weird, it seems to reformat the whole file each time you re-shrinkwrap.

Nice :+1:

File: package.json [hunk 1]
```diff
@@ -109,5 +109,8 @@
       "babel-runtime": "npm:babel-runtime@5.8.20",
       "core-js": "npm:core-js@1.1.0"
     }
+  },
+  "devDependencies": {
+    "eslint-plugin-jasmine": "^1.5.0"
```
Comment (OliverJAsh) [inline package.json]: We make all dependencies normal now, non-dev.

**Comment by jbreckmckye** [comment]
Brill :+1:

**Comment by piuccio** [comment]
would be nice to add `no-disabled-tests`

**Comment by johnduffell** [comment]
@piuccio go for it!

**Pull request merged**